installscripts/dockerfiles/jenkins/launch_jenkins_docker.sh

```diff
@@ -0,0 +1,56 @@
+#Installing Docker-ce on centos7
+sudo yum check-update
+sudo yum install -y yum-utils device-mapper-persistent-data lvm2
+sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+sudo yum install docker-ce -y
+sudo systemctl start docker
+sudo systemctl status docker
+sudo systemctl enable docker
+sudo usermod -aG docker $(whoami)
+
+#Configuring and pulling the jenkins-docker from ECR
+sudo docker volume inspect jenkins-volume
+if [ $? == 0 ] ; then
+  sudo docker volume rm jenkins-volume
+fi
+
+sudo docker volume create jenkins-volume
+aws ecr get-login --no-include-email --region us-east-1 > ecr_login_script
```

Comment:
We are trying to generalize for jazz to work in any region, can we take this a parameter?

Reply:
This is addressed in same PR.